src/updater.rs

```diff
@@ -17,6 +20,59 @@ pub struct ReleaseResponse {
     pub url: String,
 }
 
+#[derive(Eq)]
+pub struct Version(String);
+
+impl Version {
+    //Extraneous trailing zeros can throw a wrench in things
```

Comment:
I've ran into this problem with backend apis before. We can remove if we think its overkill.